### PR TITLE
[2023Q4]: Week-12 Task: Translation of "Self-introduction" Task

### PR DESCRIPTION
**Pull request by stardustmeg**
#### Title of Pull Request

Translation and Update of the Self-Introduction Video Task

#### 🤔 This is a ...

- [ ] 🌟 New task
- [ ] 🌐 New module
- [x] ⚙️ Update to an existing task
- [ ] 🔧 Update to an existing module
- [x] 🔗 Update or addition of external resources or links
- [ ] 🐛 Fix in a task or related content
- [ ] 🛠 Fix in a module or related content
- [ ] ✏️ Fixed a typo or grammatical error
- [x] 🔗 Fixed a broken link
- [ ] ❓ Other (specify: **\*\*\*\***\_\_\_\_**\*\*\*\***)

#### Description

- **Brief Overview:**

- [x] `README.md` file translated
- [x] `README.md` file style is updated to align with Module Template
- [x] broken links to RS School docs fixed
- [x] a link to a not available video resume example is removed
- [x] example repository links are updated to be relevant to the course title

#### Additional Information

- [Board task](https://github.com/rolling-scopes-school/tasks/issues/1365)

#### Checklist

- [x] ✅ I have performed a self-review of my own code.
- [ ] 📝 I have commented my code, particularly in hard-to-understand areas.
- [ ] 🔧 I have made corresponding changes to the documentation (if applicable).
- [x] 🚫 My changes generate no new warnings or errors.

### CODE REVIEW

File: stage1/modules/self-introduction/README.md [hunk 1]
```diff
@@ -1,70 +1,69 @@
...
+> - We do not require you to make the video public. You can make it [accessible only via a link](https://support.google.com/youtube/answer/157177?co=GENIE.Platform%3DDesktop&hl=en). This way only the users who receive the correct URL from you will be able to watch the video.
 
-## Проверка задания
+## Task Check
 
-Факт записи видеоролика будет проверяться с помощью [кросс-чека](https://docs.rs.school/#/cross-check-flow), а его содержание будут оценивать менторы.
+During [Cross-Check](https://docs.rs.school/#/cross-check-flow) you will have to check the fact that there is a recording, but **not** its content. The content will later be assessed by mentors.
 
-Чтобы отправить выполненное задание на проверку, зайдите в [rs app](https://app.rs.school/), перейдите на вкладку Cross-Check: Submit, выберите задание **Self-Introduction** и добавьте в форму **ссылку на деплой**.
+To submit the completed task for review, go to [RS App](https://app.rs.school/), choose the Cross-Check: Submit tab, select the **Self-Introduction** task in the dropdown menu, and add your **deployment link** to the form.
```
Comment (KristiBo) [inline stage1/modules/self-introduction/README.md:36]: Maybe move this information to the workflow?

File: stage1/modules/self-introduction/README.md [hunk 2]
```diff
@@ -1,70 +1,69 @@
...
 
-### Критерии оценки
+### Assessment Criteria
 
-Максимальный балл за задание — **100 баллов**
+The maximum score for the task is **100 points**.
 
-- Есть ссылка на видеозапись выступления. В видео автор рассказывает о себе на английском языке, видео имеет длительность около 3–5 минут (отклонение в пределах 15 секунд ошибкой считать не нужно) — **+50 баллов**
-- Есть транскрипт выступления. Транскрипт на английском языке и содержит полный текст выступления (может быть незначительно отредактирован) — **+50 баллов**
+- In the video, the author talks about themselves in English, and the video lasts for about 3-5 minutes (variation within 15 seconds must not be considered either an error or a reason to subtract any points) — **+50 points**.
```
Comment (KristiBo) [inline stage1/modules/self-introduction/README.md:44]: I think it's better not to remove this part "There is a link to the video presentation."

Comment (stardustmeg) [reply stage1/modules/self-introduction/README.md]: Done ✅

File: stage1/modules/self-introduction/README.md (hunk 1 above)
Comment (stardustmeg) [reply stage1/modules/self-introduction/README.md:36]: We could probably repeat it there too as long as we have the information about an open PR to the main branch in both `Workflow` and `Task Check`.

**Review by valerydluski — APPROVED** [review]

**Pull request merged**